app/controllers/api/v1/activities_controller.rb

```diff
@@ -6,6 +6,9 @@ def latest
 
   def just_updated
     versions = Version.just_updated(50)
+    if params[:since].present?
```

Comment:
> So maybe a simple to_i would suffice?

If there is a possibility of 500 error, it is very likely that scanners will find it and keep hammering it. I am not sure how  `params[:since].to_i` would work (`'2014-12-01'.to_i  => 2014`).
Current implementation would raise error for invalid date formats and out of range date/months.